staging/kos/pkg/controllers/ipam/ipam.go

```diff
@@ -1051,9 +1047,36 @@ func (list ParsedLockList) RemFunc(elt ParsedLock) (sans ParsedLockList, diff bo
 	return list, false
 }
 
-func FormatErrVal(err bool) string {
+func FmtErrBool(err bool) string {
 	if err {
 		return "err"
 	}
 	return "ok"
 }
+
+func SummarizeErr(err error) string {
```

Comment:
I would move this to a util pkg (to avoid duplication) under "pkg/controllers/" or "pkg/util".
But this can be addressed in a follow-up PR.

Reply:
Agree.